sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultPrivateKey.java

```diff
@@ -20,8 +20,14 @@
      */
     private static final long serialVersionUID = 30_10_00;
 
+    /**
+     * Key ID.
+     */
     private String kid;
 
+    /**
+     * Algorithm.
+     */
```

Comment:
Was this enforced by the javadoc plugin?

Reply:
Yes, I believe something changed where all fields in a serializable class need to have Javadocs

Reply:
This is a private field and requiring javadocs on private fields can become tedious. Is there a way to turn this off while generating javadocs?